earth_enterprise/src/fusion/rasterfuse/TmeshGenerator.cpp

```diff
@@ -74,14 +77,14 @@ TmeshGenerator::Generate(float *srcSamples,
   elev.resize(SizeX * SizeY);
 
   // Error thresholds for decimation - set these to 0 to skip decimation
-  // jdjohnso - These numbers came from empirical testing?
-  // jdjohnso - The allowable error gets smaller as the resolution increases.
-  // jdjohnso - The allowable error on border tiles is 1/10 that of interior
+  // Question - Did these numbers come from empirical testing?
+  // The allowable error gets smaller as the resolution increases.
+  // The allowable error on border tiles is 1/10 that of interior
   // tiles. This is because we really want the border tiles to match. If not
   // we get rips between mesh tiles in the client.
-  int error_level = tmeshAddr.level+1;
-  double error = 0.009 * pow(2, 11-error_level);
-  double borderError = error / 10.0;
+  double threshold_modifier = pow(2, 11 - (tmeshAddr.level + 1));
+  double error = decimation_threshold * threshold_modifier;
```

Comment:
decimation_threshold - double calls for higher precision than float plus everything else is double in this equation.

Reply:
its easier to understabd  11- error_level  vs 11 - (tmeshAddr.level + 1), I suggest keep as is

Reply:
We really don't need it though, do we? Wouldn't it make more sense to change everything to floats?

Reply:
So you want me to have a 4th variable just for storing (tmeshAddr.level + 1)  ?  Why?

Reply:
saw that, we can override in generate, but not sure why it cannot be kept as is in the constrcutors to initialise it there like it used to be before.

Reply:
I don't see the point in keeping it in the constructor when it isn't being used from there.  Can you explain please? We have to call Generate() with the decimation_value regardless as it is now no longer hardcoded into the Generate() function.

Reply:
I think you posted this comment in the wrong thread.